examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -0,0 +1,196 @@
+package memeland
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/seqid"
+	"std"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// Post - as previously defined
+type Post struct {
+	ID            string
+	Data          string
+	Author        std.Address
+	Timestamp     time.Time
+	Upvotes       int
+	UpvoteTracker *avl.Tree // address > bool hasVoted
```

Comment:
Why not simply have a single map that maps address -> struct{}{}?

The way it currently is, `Upvotes` will always be equal to `UpvoteTracker.Size()`

Reply:
[e59c173](https://github.com/gnolang/gno/pull/1751/commits/e59c173ac2955d6a773f2c4df8e2482566a15f27)